start-heimdall2.sh

```diff
@@ -14,25 +14,24 @@ source .env
 DATABASE_USERNAME="postgres"
 DATABASE_PASSWORD="$(openssl rand -hex 33)"
 JWT_SECRET="$(openssl rand -hex 64)"
-JWT_TIMEOUT="1d"
+JWT_EXPIRE_TIME="1d"
 API_KEY_SECRET="$(openssl rand -hex 33)"
 NAMESPACE="heimdall"
 
 # Install heimdall
-helm install heimdall heimdall2 \
+helm install heimdall heimdall2-helm/heimdall \
```

Comment:
Can you explain this change?

Reply:
Talked about this on slack/voice.  This is to make the start script match the rest of our documentation about what we recommend people name their helm chart repo.